Purchasing.Mvc/Services/NotificationService.cs

```diff
@@ -399,6 +403,10 @@ public void OrderAddAttachment(Order order, User actor)
             var users = order.OrderTrackings.Select(a => a.User).Distinct().ToList();
             foreach (var ot in users)
             {
+                if (!ot.IsActive) //Not active, we don't want to email them
```

Comment:
@srkirkland My thought is inactive users are rare, but I can change the query if you want.